### PR TITLE
Bug/heroku deployment

### PR DESCRIPTION
**Pull request by vonvick**
#### What does this PR do?
This PR was created to resolve issues with application not showing images and expected icons on Heroku.

#### Description of Task to be completed?
No task

#### How should this be manually tested?
The PR can be tested during deployment to Heroku.

#### Any background context you want to provide?
No background context

#### What are the relevant pivotal tracker stories?
No relevant pivotal tracker story

#### Screenshots (if appropriate)
Not Applicable

#### Questions:
Not Applicable.

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,2 @@
+MONGOHQ_URL=mongodb://{username}:{port}/{databasename}
+PORT=xxxx
```
Comment (oagboola) [inline .env.sample]: why is 'xxxx' the value of PORT here?

File: .eslintignore [hunk 2]
```diff
@@ -4,3 +4,4 @@
 ./*
 !app/*
 
+
```
Comment (oagboola) [inline .eslintignore]: what's this extra line added for?

File: server.js [hunk 3]
```diff
@@ -22,7 +22,7 @@ var env = process.env.NODE_ENV = process.env.NODE_ENV || 'development',
     mongoose = require('mongoose');
 
 //Bootstrap db connection
-console.log("This is the database url", config.db);
+console.log("This is the database url ", config.db);
```
Comment (oagboola) [inline server.js]: This is not needed anymore, get rid of this

File: gulpfile.js [hunk 4]
```diff
@@ -1,81 +1,84 @@
...
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const nodemon = require('gulp-nodemon');
+const bower = require('gulp-bower');
+const browserSync = require('browser-sync');
+const istanbul = require('gulp-istanbul');
+const eslint = require('gulp-eslint');
+const filter = require('gulp-filter');
+const plugins = require('gulp-load-plugins')();
```
[the remark this answers is not in the document]
Comment (oagboola) [reply gulpfile.js:10]: @andela-vnwaiwu check this out

File: gulpfile.js [hunk 5]
```diff
@@ -1,81 +1,84 @@
...
-var filter = require('gulp-filter');
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const nodemon = require('gulp-nodemon');
+const bower = require('gulp-bower');
+const browserSync = require('browser-sync');
+const istanbul = require('gulp-istanbul');
+const eslint = require('gulp-eslint');
+const filter = require('gulp-filter');
```
[the remark this answers is not in the document]
Comment (oagboola) [reply gulpfile.js:9]: @andela-vnwaiwu and this too

**Comment by oagboola** [comment]
check all hound ci comments on unresolved modules

**Comment by vonvick** [comment]
The unresolved modules are as a result of installing the above-mentioned modules as devDependencies and not in Dependencies. I believe If they are moved to Dependencies they would not cause HoundCI errors. I feel they are installed in devDependencies because they are for development purposes. Should I move them to Dependencies?

**Comment by oagboola** [comment]
yes, do that

**Pull request closed without merging**